internal/node/node.go

```diff
@@ -59,6 +60,12 @@ type AutopilotConfig struct {
 
 type ShutdownFn = func(context.Context) error
 
+var mainnet *consensus.Network
+
+func init() {
```

Comment:
idk why I suggested using `init` for this, I must have been sleep deprived or something lmao. Instead you should add a `network` field to `chainManager` and initialize it to `mainnet` (or testnet, or whatever) in `NewBus`